ASP.NET_Core/FoodOrdering.API/Controllers/ProductsController.cs

```diff
@@ -11,27 +12,85 @@ namespace FoodOrdering.API.Controllers
     public class ProductsController : ControllerBase
     {
         private readonly IUnitOfWork _unitOfWork;
-        private readonly ILogger<ProductsController> _logger;
 
-        public ProductsController(ILogger<ProductsController> logger, IUnitOfWork unitOfWork)
+        public ProductsController(IUnitOfWork unitOfWork)
```

Comment:
use service from BLL in controller
in service use IUnitOfWork 
remove IUnitOfWork  from controller